### DISCUSSION TITLE
VOD uploaded to S3 but link in VOD page not change

### DISCUSSION THREAD

**Discussion by leekung** [ant-media/Ant-Media-Server#5871] [Q&A] [answered]
I have set Stream Recording to mp4 and S3 recording (DigitalOcean Space)
The VOD was saved and uploaded to S3 bucket as it should be

In server, the file already deleted from file system (webapps/WebRTCAppEE/streams/)
but in the VOD list tab, the VOD file still link to local file, cause to 404, cannot play vod file

I think the link in VOD tab should change to S3 URL. Is not it?

Thanks

**Comment by yashtandon113** [comment, marked as answer]
@leekung 
Yes, you are right. For that, you have to enable HTTP forwarding on your server, so after the VOD is uploaded, your request will be forwarded to the bucket. Please check out the following document:
https://antmedia.io/docs/guides/recording-live-streams/s3-integration-http-forwarding/#http-forwarding

If you are using the latest version 2.6.2 and above of AMS, you can change these settings from the web panel itself now. Please check out the following [discussion comment](https://github.com/orgs/ant-media/discussions/5161#discussioncomment-6401677). 

I hope it helps.

**Reply by leekung** [reply]
Thanks, I missed that topic. It works now.

Note: The s3Permission should be "public-read" for this to work
